packages/react-dogfood/components/Stage.tsx

```diff
@@ -9,23 +9,17 @@ export const Stage = ({
 }) => {
   const call = useCall();
   const searchParams = useSearchParams();
-
   const groupSize = +(searchParams.get('group_size') ?? '0');
 
   const SelectedComponent = LayoutMap[selectedLayout].Component;
 
-  if (selectedLayout === 'LegacyGrid' || selectedLayout === 'LegacySpeaker') {
-    return (
-      <div className="str-video__stage">
-        <SelectedComponent call={call!} />
-      </div>
-    );
-  }
-
   return (
-    // @ts-expect-error
-    <SelectedComponent
-      groupSize={!groupSize || groupSize > 16 ? undefined : groupSize}
-    />
+    <div className="str-video__stage">
+      <PermissionRequests />
```

Comment:
Let's move this to MeetingUI above Stage instead of having it within Stage directly. We should also rename Stage to something like `ActiveCallView` or something (but that's for another PR).

Reply:
done.